lib/eofs/tests/test_solution.py

```diff
@@ -276,6 +276,37 @@ def _tomasked(self, value):
         return ma.MaskedArray(value, mask=np.isnan(value))
 
 
+class TestStandardDask(StandardSolutionTest):
+    weights = 'equal'
+
+    def _tomasked(self, value):
+        return ma.masked_invalid(value)
+
+    @classmethod
+    def modify_solution(cls):
+        # Skip this class if dask not available
+        pytest.importorskip('dask.array', minversion='1.0')
```

Comment:
I don't know that this works correctly. I'm seeing skipped dask tests in the test output:
```
SKIPPED [30] /home/travis/build/ajdawson/eofs/lib/eofs/tests/test_solution.py:289: module 'dask.array' has __version__ None, required is: '1.0'
```
Should this just be `'dask'` instead of `'dask.array'`.

Reply:
Thanks for spotting - using `dask.array` is intentional - dask can be installed with only some of its components, so this ensures the array component is installed.

There is a problem with checking the version here though - the version of Dask being used by the old Python 3.4 tests had a different syntax so I wanted to disable tests for that version. `dask.array` doesn't have a `__version__` attribute though (that's part of the `dask` namespace), so the tests got skipped all the time.

Since the Python 3.4 tests are no longer being used I've removed the version constraint. There are now no skipped tests in any of the 'extras' test runs